static/schemas/2.0/workflowGraphs.2.0.json

```diff
@@ -0,0 +1,59 @@
+{
+    "title": "workflowGraphs.2.0",
+    "definitions": {
+        "tasks": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "additionalProperties": false,
+                "oneOf": [
+                    {"required": ["taskName"]},
+                    {"required": ["taskDefinition"]}
```

Comment:
I don't think the graph instance can be created successfully if without label, just search the `label` in [task-graph.js](https://github.com/RackHD/on-tasks/blob/master/lib/task-graph.js) will get the result. So the doc is not correct.

Anyway, add @benbp to double confirm.